packages/stark-ui/src/modules/table/components/table.component.html

```diff
@@ -53,7 +52,15 @@
 					</button>
 				</div>
 			</mat-menu>
-		</div>
+		</ng-container>
+		<stark-minimap
+			[matTooltip]="'STARK.TABLE.TOGGLE_COLUMNS' | translate"
```

Comment:
I think the translation we have for this key is not correct... it says "Column filters" but it is not to filter columns. I suggest to change it to "Show/Hide columns".

I think the same translation key is used in the `column.component.html` but in that case the "Column filter" does make sense... so we are definitly missing one translation key

Reply:
Updated the PR. 😄